plugins/javaembed/javaembedOptions.md

```diff
@@ -0,0 +1,46 @@
+# Java Options for HPCC
+
+The embedded Java plugin is managed through several configuration options that control its runtime behavior. These settings allow you to customize how the plugin interacts with the Java Virtual Machine (JVM), manages native libraries, and locates Java classes and dependencies.
+
+## Containerized Deployments
+
+- **JAVA_LIBRARY_PATH**
+  - Environment variable.
+  - Sets the `-Djava.library.path` JVM option.
+  - Specifies where the JVM should look for native libraries (such as `.so`, `.dll`, or `.dylib` files) used by Java code via the Java Native Interface (JNI).
+- **JVM_OPTIONS**
+  - Environment variable.
+  - Space-delimited list of JVM options appended to the JVM invocation.
+  - Controls the behavior of the JVM when launched from ECL.
+- **CLASSPATH**
+  - Environment variable.
+  - List of directories, JAR files, or ZIP files that the JVM searches when loading classes not included in the Java standard library.
+  - Default: `hpccBuildInfo.installDir).append(PATHSEPCHAR).append("classes")`
+  - Useful for specifying the location of target Java classes and their dependencies.
+
+In containerized environments, you can only use Java versions included in the HPCC Systems-provided image.  
+To use a different Java version or distribution, you must customize the base Docker image and set the above configuration options accordingly.
```

Comment:
if there's guidance on how to create a custom image, we should link to it here. @GordonSmith might have knowledge about this.